src/MessageBird/Client.php

```diff
@@ -160,11 +163,11 @@ class Client
      * @param string            $accessKey
      * @param Common\HttpClient $httpClient
      */
-    public function __construct($accessKey = null, Common\HttpClient $httpClient = null)
+    public function __construct($accessKey = null, Common\HttpClient $httpClient = null, Array $config = [])
```

Comment:
Could you use `array` instead of `Array`, it's an internal PHP type.

Reply:
I'll change it to make it consistent